jans-cedarling/test_files/policy-store-with-default-entities.yaml

```
@@ -145,6 +145,27 @@ policy_stores:
+          "entity_id": "1694c954f8d9",
+          "entity_type": "Jans::DefaultEntity",
+          "o": "Acme Dolphins Division",
+          "org_id": "100129",
+          "domain": "acme-dolphin.sea",
+          "regions": ["Atlantic", "Pacific", "Indian"],
+        }
+      "74d109b20248":
+        {
+          "entity_type": "Jans::DefaultEntity",
+          "description": "2025 Price List",
+          "products": { "15020": 995, "15050": 1495 },
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)